sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -1839,7 +1840,18 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       }.getMessage
       assert(msg.contains("Find an invalid url string"))
     }
-    withSQLConf(SQLConf.ANSI_ENABLED.key -> "false") {
+    withSQLConf(SQLConf.ANSI_ENABLED.key -> "true") {
+            checkError(exception = intercept[SparkIllegalArgumentException]{
+              evaluateWithoutCodegen(
+             ParseUrl(Seq("https://a.b.c/index.php?params1=a|b&params2=x", "HOST")))
```

Comment:
Can we make `"https://a.b.c/index.php?params1=a|b&params2=x"` as variable and reuse here and there?

Reply:
Done.